man/man1/kbsecret-session.1.ronnpp

```diff
@@ -0,0 +1,90 @@
+kbsecret-session(1) - create a new kbsecret(1) session
+=======================================================
+
+## SYNOPSIS
+
+`kbsecret session new` [options] <label>
+`kbsecret session rm` [options] <session>
+
+## DESCRIPTION
+
+Creates or deletes kbsecret sessions.
+
+## COMMANDS
+
+* `new`:
+	Creates a new session.
+
+	There are two kinds of sessions: *team-based* and *teamless* (the default).
+
+	*Team-based* sessions are placed in a team folder managed by Keybase. Teams have access controls
+	and a mutable user list, making them ideal for sharing secrets between members of a changing
+	team.
+
+	*Teamless* sessions are placed in a folder managed by KBSecret itself. Teamless sessions
+	have a fixed number of users, all of whom have full read/write access to all secrets. Teamless
+	sessions are ideal for isolating one's own secrets (single-user sessions).
+
+	The following options are permitted for the `new` subcommand:
+
+	* `-t`, `--team` <team>:
```

Comment:
Instead of using a nested list here (`ronn(1)` renders them incorrectly), write valid options out in prose form (see `kbsecret-generator(1)` for inspiration).